lib/bootstrap.js

```diff
@@ -0,0 +1,95 @@
+import UiAutomator from 'appium-uiautomator';
+import ADB from 'appium-adb';
+import log from './logger';
+import net from 'net';
+import path from 'path';
+import _ from 'lodash';
+import { errorFromCode } from 'mobile-json-wire-protocol';
+
+class AndroidBootstrap {
+  constructor (systemPort, webSocket) {
+    this.systemPort = systemPort;
+    this.webSocket = webSocket;
+  }
+
+  async start (appPackage, disableAndroidWatchers) {
+    try {
+      let rootDir = path.resolve(__dirname, '../..');
+      let startDetector = (s) => { return /Appium Socket Server Ready/.test(s); };
+      const bootstrapJar = path.resolve(rootDir, 'bootstrap', 'bin', 'AppiumBootstrap.jar');
```

Comment:
What is the difference between `bootstrapJar`, and `rootDir` and `startDetector`, that the first is `const` and the others aren't?